examples/car_control/run.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+
+import numpy as np
+import os
+import spear
+import time
+
+common_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "common"))
+import sys
+
+sys.path.append(common_dir)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--actions_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.csv")))
+    parser.add_argument("--bp_class_reference", default="/Game/VehicleTemplate/Blueprints/OffroadCar/OffroadCar_Pawn.OffroadCar_Pawn_C")
```

Comment:
For simplicity, no need for `bp_class_reference` to be an arg.

Reply:
removed.